src/resources/js/commerce/gateway/paypal/checkout.js

```diff
@@ -540,10 +541,17 @@ tribe.tickets.commerce.gateway.paypal.checkout = {};
 	 * Handle actions when checkout buttons are loaded.
 	 *
 	 * @since 5.1.10
+	 * @since TBD Replaced DOMNodeInserted with animationstart event.
 	 */
-	obj.buttonsLoaded = function () {
+	obj.buttonsLoaded = function ( event ) {
+		if ( event.originalEvent.animationName !== 'node_inserted' ) {
+			return;
+		}
+
 		$document.trigger( tribe.tickets.commerce.customEvents.hideLoader );
-		$( tribe.tickets.commerce.selectors.checkoutContainer ).off( 'DOMNodeInserted', obj.selectors.buttons, obj.buttonsLoaded );
+		$( tribe.tickets.commerce.selectors.checkoutContainer ).off( 'animationstart', obj.selectors.animatedElement, obj.buttonsLoaded );
+		$( tribe.tickets.commerce.selectors.checkoutContainer ).off( 'MSAnimationStart', obj.selectors.animatedElement, obj.buttonsLoaded );
+		$( tribe.tickets.commerce.selectors.checkoutContainer ).off( 'webkitAnimationStart', obj.selectors.animatedElement, obj.buttonsLoaded );
```

Comment:
I wonder if the on/off is large enough to warrant it's own toggle function 🤔 (not a request, just a thing to think about)

Reply:
Also wonder if the browser-prefixes are necessary? The [_element API_](https://caniuse.com/?search=animationstart) for  animationstart has nearly 97% and all browsers since 2020 are good with it.

Reply:
Ditto for prefixed @keyframes and the animation-duration prop (below)

Reply:
They might not be, but i preferred to leave them there just in case. I wasnt able to test this in all modern browsers. Happy to follow your lead on this one!

Regarding creating methods for the on/off for now i think its an overkill. If we would need to do it one more or if there were more events to on/off i think it would be worth it.